origin/origin_ui.go

```diff
@@ -161,10 +161,32 @@ func handleExports(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, res)
 }
 
+// Create a new private key in the given directory, but without loading it immediately
+// This new key will be detected and loaded later by the ongoing goroutine `LaunchIssuerKeysDirRefresh`,
+// which periodically refreshes the keys in the issuer keys directory
+func createNewIssuerKey(ctx *gin.Context) {
+	issuerKeysDir := param.IssuerKeysDirectory.GetString()
```

Comment:
My on the return still stands, but looking at the original issue, generating new private keys is specifically not in scope got this issue.